docs/master/api-reference/directives.md

```diff
@@ -321,22 +323,53 @@ type User {
 }
 ```
 
-When using the `type` argument with pagination style `CONNECTION`, you may create your own
-[Edge type](https://facebook.github.io/relay/graphql/connections.htm#sec-Edge-Types) which
-may have fields that resolve from the model [pivot](https://laravel.com/docs/eloquent-relationships#many-to-many)
-data. You may also add a custom field resolver for fields you want to resolve yourself.
+### Retrieving Intermediate Table Columns
+
+You may want to allow accessing data that describes the relation between the models
+and is stored in the intermediate table - see [retrieving intermediate table columns in Laravel](https://laravel.com/docs/eloquent-relationships#retrieving-intermediate-table-columns).
+
+Just like in Laravel, you can access the `pivot` attribute on the models (or its alias).
+Even though this attribute is always present when querying the model through the relation,
+it may not be present when reaching the node through another path in the schema, so it is
+recommended to define the field as nullable (no `!`).
+
```

Comment:
In the following example we have two models `User` and `Role` connected via a `RoleUserPivot`. The `RoleUserPivot` contains an additional attribute `meta`.